shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -64,6 +70,20 @@ @interface FlutterViewController () <FlutterBinaryMessenger, UIScrollViewDelegat
 @property(nonatomic, assign) double targetViewInsetBottom;
 @property(nonatomic, retain) CADisplayLink* displayLink;
 
+/*
+ * Trackpad gesture recognizers
```

Comment:
It's for other hardware (mouse wheel) as well. Probably I should change this comment.

Reply:
Can you explicitly call out which ones are for trackpad and which ones are for mouse, etc. It will greatly improve readability and code review. Thanks